editor/plugin_config_dialog.cpp

```diff
@@ -215,11 +219,15 @@ PluginConfigDialog::PluginConfigDialog() {
 	grid->add_child(script_option_lb);
 
 	script_option_edit = memnew(OptionButton);
+	int default_lang = 0;
 	for (int i = 0; i < ScriptServer::get_language_count(); i++) {
-		ScriptLanguage *lang = ScriptServer::get_language(i);
-		script_option_edit->add_item(lang->get_name());
+		String lang = ScriptServer::get_language(i)->get_name();
+		script_option_edit->add_item(lang);
+		if (lang == GDScriptLanguage::get_singleton()->get_name()) {
```

Comment:
A better way to check the language would be:

``` C++
ScriptLanguage *lang = ScriptServer::get_language(i);
script_option_edit->add_item(lang->get_name());
if (lang == GDScriptLanguage::get_singleton()) {
	// ...
```

Reply:
@neikeq Changed. I copied the code from ScriptCreateDialog which actually has a hardcoded `lang == "GDScript"` check.